controllers/jest.setup.js

```diff
@@ -0,0 +1 @@
+jest.setTimeout(10000);
```

Comment:
Won't this affect the tests on other branches/future tests?

I.e. will all tests now be held up due to this timeout?